### PR TITLE
Enable/Disable Remote Login

### PR DESCRIPTION
**Pull request by thomanq**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -958,6 +958,18 @@ ping -o github.com
 traceroute github.com
 ```
 
+### SSH
+
+#### Remote Login
+```bash
+# enable remote login
```
Comment (herrbischoff) [inline README.md]: Please capitalize at the start.

File: README.md [hunk 2]
```diff
@@ -958,6 +958,18 @@ ping -o github.com
...
+sudo launchctl load -w /System/Library/LaunchDaemons/ssh.plist
+
+# disable remote login
```
Comment (herrbischoff) [inline README.md]: Please capitalize at the start.

**Comment by herrbischoff** [comment]
Please see my inline comments. Also, please add the SSH sub-category to the index on top.

**Comment by thomanq** [comment]
Done.

**Pull request merged**